.github/docker-images/swift-5-centos-x64/Dockerfile

```diff
@@ -10,34 +10,11 @@ RUN yum -y update \
     sudo \
     # Python
     python3 \
-    python3-devel \
     python3-pip \
-    make \
-    cmake \
-    cmake3 \
     openssl-devel \
-    gcc-c++ \
     && yum clean all \
-    && rm -rf /var/cache/yum 
-###############################################################################
-# Handle cmake on centos properly
-###############################################################################
-RUN sudo alternatives --install /usr/local/bin/cmake cmake /usr/bin/cmake 10 \
```

Comment:
wasnt sure if any of this was needed for the builder, but glad to see it's not :)